restconnector/RequestHandler.cpp

```diff
@@ -83,7 +95,6 @@ void RequestHandler::processGET_( const ::zeq::Event& event )
 
     while( blocked_ )
     {
-        while( subscriber_->receive( 0 ) ){}
```

Comment:
This thread is blocked until the listening thread receive a rsponse and set "blocked_" to false.